tiledb/sm/cpp_api/query_experimental.h

```diff
@@ -241,6 +241,11 @@ class QueryExperimental {
    * Returns the number of elements in the result buffers from a read query.
    * This is a map from the attribute name to a pair of values.
    *
+   * For fixed size dimension labels the non-experimental API will return the
```

Comment:
I would remove this and modify the first paragraph slightly:   
* Returns the number of elements for dimension labels in the result buffers from a read query.
* This is a map from the attribute name to a pair of values.

Also look at the rest of the text and replace attribute(s) by label(s) and remove any reference to coordinates.

Reply:
Done, thanks!